src/module.ts

```diff
@@ -97,6 +97,11 @@ export const plugin = getPanelPluginOrFallback(
             showIf: (config: HeatmapFieldConfig) =>
               config.colorPalette !== 'custom' && config.colorPalette !== 'fieldOptions',
           })
+          .addColorPicker({
+            path: 'nullValueColor',
+            name: 'Null Value Color',
```

Comment:
```suggestion
            name: 'Null value color',
```

For consistency; Other options are in sentence case. Also, I'm wondering if the name alone is clear enough or whether it warrants a description. We could also wait to collect feedback.